app/Authentication/routes/authentication-routes.php

```diff
@@ -31,4 +32,6 @@
     /**SOCALITE AUTH */
     Route::get('/login/{provider}/redirect', [SocialiteAuthController::class, 'redirect'])->withoutMiddleware(['auth:sanctum']);
     Route::get('/login/{provider}/callback', [SocialiteAuthController::class, 'handleCallback'])->withoutMiddleware(['auth:sanctum']);
+    Route::get('/register/{provider}/redirect', [MemberSocialAuthController::class, 'redirect'])->withoutMiddleware(['auth:sanctum']);
+    Route::get('/register/{provider}/callback', [MemberSocialAuthController::class, 'handleCallback'])->withoutMiddleware(['auth:sanctum']);
```

Comment:
We should move these routes and code to the platform module since it's related to registration and not authentication.